Steps to build from source seems to be incomplete

From the README.md:

> To build from source you can:
> * Clone this repository and change directory into it
> * Run `make bootstrap`
>
> This will leave you with `./glide`, which you can put in your `$PATH` if you'd like. (You can also take a look > at `make install` to install for you.)

When I ran the following steps:

```shell
git clone https://github.com/Masterminds/glide.git
cd glide
make bootstrap
```

I ended up with no `./glide` file.

Looking at the makefile, I assumed that `make` or `make build` would work. However, both fails with this output:

```
go build -o glide -ldflags "-X main.version=0.7.0" glide.go
glide.go:42:2: cannot find package "github.com/Masterminds/cookoo" in any of:
	/home/jeannikl/code/.go1.5/src/github.com/Masterminds/cookoo (from $GOROOT)
	/home/jeannikl/code/golang/src/github.com/Masterminds/cookoo (from $GOPATH)
glide.go:40:2: cannot find package "github.com/Masterminds/glide/cmd" in any of:
	/home/jeannikl/code/.go1.5/src/github.com/Masterminds/glide/cmd (from $GOROOT)
	/home/jeannikl/code/golang/src/github.com/Masterminds/glide/cmd (from $GOPATH)
glide.go:43:2: cannot find package "github.com/codegangsta/cli" in any of:
	/home/jeannikl/code/.go1.5/src/github.com/codegangsta/cli (from $GOROOT)
	/home/jeannikl/code/golang/src/github.com/codegangsta/cli (from $GOPATH)
Makefile:5: recipe for target 'build' failed
make: *** [build] Error 1
```

This happens both with and without GO15VENDOREXPERIMENT set.

I am running `go version go1.5 linux/amd64`.

Not sure if this is an error with my setup or an error with the Readme/Makefile, but help would be appreciated either way.

--- a/README.md
+++ b/README.md
@@ -74,8 +74,11 @@ $ brew install glide
 
 To build from source you can:
 
-1. Clone this repository and change directory into it
-2. Run `make bootstrap`
+1. Clone this repository into `$GOPATH/src/github.com/Masterminds/glide` and
+   change directory into it
+2. Ensure that the environment variable GO15VENDOREXPERIMENT is set, f.ex. by
+   running `export GO15VENDOREXPERIMENT=1`
+3. Run `make bootstrap`, followed by `make build`
 
 This will leave you with `./glide`, which you can put in your `$PATH` if
 you'd like. (You can also take a look at `make install` to install for

